content/applications/general/payment_acquirers/adyen.rst

```diff
@@ -99,5 +99,27 @@ where payments will be made (the URLs of the servers hosting your Odoo instances
    :align: center
    :alt: Allows payments originated from a specific domain.
 
+Capture payments after the delivery
+-----------------------------------
+
+Adyen allows you to capture an amount manually instead of having an immediate capture.
+
+To set it up, enable the **Capture Amount Manually** on Odoo, as explained in the :ref:`payment
+acquirers documentation <payment_acquirers/capture_amount>`, then on Adyen.
+
+To do so, open your Adyen Merchant Account, go to :menuselection:`Account --> Settings`, and set the
+**Capture Delay** to **manual**.
+
+.. image:: media/adyen_capture_delay.png
+   :align: center
+   :alt: Capture Delay settings in Adyen
+
+.. warning::
+   * If you set the **Capture Amount Manually** on Odoo but not on Adyen, Odoo can't detect it.
+     Therefore, your authorization request would be automatically approved in an immediate capture
+     on Adyen's side.
```

Comment:
Although correct, it's a rather technical explanation. In the end, all the user needs to know is "proper configuration = good; otherwise bad".
```suggestion
   * If the **Capture Delay** is not set to **manual** on Adyen, Odoo will ignore the configuration option **Capture Amount Manually**.
```
Please correct if it's not what will actually happen on Odoo.

Reply:
That's False, Odoo will not ignore it, and since the `Authorisation` notification will be exactly the same, Odoo will put the transaction as `authorized` where Adyen will consider it `done` and never touch it again. And so, Odoo will wait... and wait... and wait... And every capture or void request will just return with a `success: false`. That's why I put an emphasis here. This'll break every tx if not done correctly.

Reply:
Ok. Then,
1. Make it a `danger` admonition.
2. Change the line to something like "If the **Capture Delay** is not set to **manual** on Adyen but Odoo is not configured to capture amounts manually, it will be impossible to capture or cancel payments from Odoo."
3. Split the second line into another admonition, a `note` should be fine IMO.
4. Use dashes (`-`) for bullet points.

Reply:
I may have misread this the first time because I didn't understand the _dangerous_ aspect of it. A good admonition would work marvelously here indeed :)